backend/app/controllers/backend/albums_controller.rb

```diff
@@ -87,7 +87,7 @@ def set_album
       end
 
       def set_albums
-        @albums = Album.albums(@search, @index_items_limit * @page)
+        @albums = Album.albums(params, @index_items_limit * @page)
```

Comment:
It's similar to what we do here: https://github.com/Vizzuality/grid-arendal/blob/master/backend/app/controllers/backend/albums_controller.rb#L80